simple-chat-app/src/app/components/chats/chats.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="host">
+  <div class="contener" *ngIf="selectedUser; else loading">
+    <h2>{{ selectedUser.name }}</h2>
+
+    <div class="messages">
+      <div
+        [ngClass]="{ external: msg.from !== 'me' }"
+        class="msg"
+        *ngFor="let msg of messages"
+      >
+        <div *ngIf="msg.from !== 'me'" class="external-icon">
+          {{ getFirstLetters(selectedUser.name) }}
```

Comment:
never use get functions inside html either map the data inside the service or create a new pipe